pandas/tests/tseries/test_timezones.py

```diff
@@ -65,6 +65,145 @@ def cmptz(self, tz1, tz2):
         # tests.
         return tz1.zone == tz2.zone
 
+
```

Comment:
I think you need to audit what you are moving here. A lot of these are fairly generic routines for series/dataframe that are actually testing the operation in question, and NOT the timezone per-se. Moving them here is really confusing.